pom.xml

```diff
@@ -275,6 +271,33 @@ under the License.
       <artifactId>maven-plugin-annotations</artifactId>
     </dependency>
 
+    <!-- aether -->
+    <dependency>
+      <groupId>org.eclipse.aether</groupId>
```

Comment:
This is still an old version. You want maven-resolver:

<dependency>
    <groupId>org.apache.maven.resolver</groupId>
    <artifactId>maven-resolver-api</artifactId>
    <version>1.4.2</version>
</dependency>